cinema/models.py

```diff
@@ -84,40 +84,24 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
+    @staticmethod
+    def validate_position(value: int, max_value: int, field_name: str, error_to_raise):
+        if not (1 <= value <= max_value):
+            raise error_to_raise({
+                field_name: f"{field_name} must be in the range [1, {max_value}]"
+            })
```

Comment:
The `validate_position` method is correctly implemented as a static method to validate the row and seat positions. Ensure that the `ValidationError` is imported from `django.core.exceptions` to avoid any runtime errors.